src/TypeScript/Analyzer/JavaScriptAnalyzer.cs

```diff
@@ -218,6 +225,11 @@ private void ActiveSolutionTracker_ActiveSolutionChanged(object sender, ActiveSo
             StopServer();
         }
 
+        private void AnalysisConfigMonitor_ConfigChanged(object sender, EventArgs e)
+        {
+            StopServer();
```

Comment:
Minor logging issue: our "server started" logs are written always, but the "stopped" logs are debug only. Do you think it's worth changing? I had to turn off my debug logs due to noise and then didn't understand why I wasn't seeing any stop logs.

Reply:
It would be better to be consistent, and it does look weird only getting "start" messages. I'd make it non-debug.
Also, do we log the process id? Knowing it makes it easier to check e.g. memory usage.

Reply:
Yep we log it. I'll open a PR for the non debug then.